cocos/platform/CCFileUtils.cpp

```diff
@@ -42,6 +42,8 @@ THE SOFTWARE.
 #endif
 #include <sys/stat.h>
 
+#define DECLARE_GUARD std::lock_guard<std::recursive_mutex> __guard__(_mutex)
```

Comment:
Identifiers should begen from _ and __ .
17.4.3.1.2 Global names [lib.global.names]
Certain sets of names and function signatures are always reserved to the implementation:

Each name that contains a double underscore (__) or begins with an underscore followed by an uppercase letter (2.11) is reserved to the implementation for any use.
Each name that begins with an underscore is reserved to the implementation for use as a name in the global namespace.165